packages/live-preview-sdk/src/index.ts

```diff
@@ -3,6 +3,8 @@ import './styles.css';
 import { type DocumentNode } from 'graphql';
 
 import { version } from '../package.json';
+import { encodeSourceMap as CSM } from './csm';
```

Comment:
why the renaming? I think encodeSourceMap makes sense no? Maybe rename to `encodeCSM`? I think we are like to export more functions from the `./csm` file in future

Reply:
renaming wasn't necessary you're right 👍